python/data-wrangling-components/data_wrangling_components/engine/verbs/onehot.py

```diff
@@ -3,16 +3,34 @@
 # Licensed under the MIT license. See LICENSE file in the project.
 #
 
+from typing import Dict, List

```

Comment:
Please make sure that you use `black` and `isort` 

`poetry run black data_wrangling_components/`
`poetry run isort data_wrangling_components/`